### ISSUE TITLE
[JavaScript] Division operator and regex confusion

### ISSUE THREAD

**Issue by wbond** [sublimehq/Packages#204]
The following example is not highlighting properly with the revised syntax:

```js
width/2 + lineStart * Math.sin(i * 30 * π/180)
```

**Comment by wbond** [comment]
This should be resolved by eda3e39fb0e5667c3e4322a0f7ce885322106465

**Comment by websash** [comment]
Ok, now we have an issue in this line
```javascript
var angle = 2*π / count // angle between circles
```
`/ count /` is being treated as regex. This is in https://github.com/sublimehq/Packages/commit/0de86593947458808d394a1d5a53ceaac95515d7
